### PR TITLE
update devel/inspect-screen for the new world order

### PR DESCRIPTION
**Pull request by myk002**
image
image
image

### CODE REVIEW

File: devel/inspect-screen.lua [hunk 1]
```diff
@@ -255,7 +255,14 @@ local function get_map_report(show_empty)
     local report = {}
 
     local pos = cur_mouse_pos
-    if not pos or pos.x < 0 or pos.y < 0 or not pos.z or pos.z < 0 then
+    if not pos or not pos.z then
```
Comment (lethosor) [inline devel/inspect-screen.lua:258]: Do we not need the `< 0` checks anymore?

Comment (myk002) [reply devel/inspect-screen.lua:258]: No, I fixed the bug in Gui::getMousePos() that was allowing that condition to occur in the first place

Comment (lethosor) [reply devel/inspect-screen.lua:258]: Oh, I think I see, the `gui_getMousePos()` wrapper in LuaApi should be returning nil for invalid coordinates. Guessing the fix is in https://github.com/DFHack/dfhack/pull/2531/files#diff-a3cdb8e239e182d28f7a044fde4b571eac34f81293c8b471d43d512b129c0602R2152

Comment (myk002) [reply devel/inspect-screen.lua:258]: that's right

**Comment by lethosor** [comment]
I'm assuming, like https://github.com/DFHack/scripts/pull/497#issuecomment-1367750155, that https://github.com/DFHack/dfhack/pull/2531 is required for the tile display to work, so I'm interested in testing this tomorrow.

**Review by lethosor — APPROVED** [review]
(Actually, that built much faster than I expected!)

**Comment by myk002** [comment]
> I'm assuming, like [#497 (comment)](https://github.com/DFHack/scripts/pull/497#issuecomment-1367750155), that [DFHack/dfhack#2531](https://github.com/DFHack/dfhack/pull/2531) is required for the tile display to work, so I'm interested in testing this tomorrow.

Yes, DFHack/dfhack#2531 provided the Core functionality for this to work.

**Comment by myk002** [comment]
> (Actually, that built much faster than I expected!)

ha, yeah, no header changes no cache eviction

**Comment by lethosor** [comment]
`changelog.txt` pls

and I suspect your wineserver tweaks have helped too :)

**Pull request merged**